src/components/Layout/Header/NavBar/DrawerWallet.tsx

```diff
@@ -119,13 +122,14 @@ const AccountTableSkeleton: FC = memo(() => (
   </Stack>
 ))
 
-export const DrawerWallet: FC = memo(() => {
+const DrawerWalletInner: FC = memo(() => {
```

Comment:
Now inner - current `<DrawerWallet />` wrapped in `<MemoryRouter />` to do well, memory routing!